src/java.desktop/share/classes/java/awt/Cursor.java

```diff
@@ -300,16 +300,15 @@ public static Cursor getSystemCustomCursor(final String name)
             String prefix = CURSOR_DOT_PREFIX + name;
             String key    = prefix + DOT_FILE_SUFFIX;
```

Comment:
After you removed `containsKey(key)`, `key` is used only once, and `prefix + DOT_FILE_SUFFIX` can be inline in the call to `systemCustomCursorProperties.getProperty` to get the file name. This would align with other usages of `prefix` in the method.